localapiclient/localapiclient.go

```diff
@@ -0,0 +1,81 @@
+package localapiclient
```

Comment:
It looks like you've defined LocalAPIResponseWriter twice.  Is that necessary?  (seems obtuse, but maybe go requires it)  Should this just be "type LocalApi interface" and it's implementation?

Reply:
Oops. This was leftover from a previous attempt. I'm bad at git.